src/administration/integrations/gluescript.md

```diff
@@ -0,0 +1,251 @@
+# GlueScript
+
+GlueScript scripts are powered by a lightweight Javascript engine that allows you to run a script on any activity within Platform.sh. This opens a whole world of new automation use-cases including, but not limited to, calling external web services.
```

Comment:
It will be incorrect, these are not only user-initiated actions (ie this would happen on automated snapshots... but also a reaction to PRs etc..). So activity seems fine and it is the terminology we use in other places. You can propose longer text...